content/docs/reference-glossary.md

```diff
@@ -1,43 +1,43 @@
 ---
 id: glossary
-title: Glossary of React Terms
+title: React की शर्तों का शब्दकोष
 layout: docs
 category: Reference
 permalink: docs/glossary.html
 
 ---
 
-## Single-page Application {#single-page-application}
+## सिंगल-पेज एप्लीकेशन {#single-page-application}
 
-A single-page application is an application that loads a single HTML page and all the necessary assets (such as JavaScript and CSS) required for the application to run. Any interactions with the page or subsequent pages do not require a round trip to the server which means the page is not reloaded.
+सिंगल-पेज एप्लिकेशन एक ऐसा एप्लिकेशन है जो एक HTML पेज को लोड करता है और आवश्यक सभी संपत्तियो को भी (जैसे जावास्क्रिप्ट और सीएसएस) जो एप्लिकेशन को चलाने के लिए आवश्यक है। पृष्ठ या बाद के पृष्ठों के साथ किसी भी बातचीत को सर्वर के लिए एक गोल यात्रा की आवश्यकता नहीं होती है जिसका अर्थ है कि पृष्ठ पुनः लोड नहीं हुआ है।
```

Comment:
assets should be assets in hindi

Reply:
सीएसएस can be CSS, no need to translate

Reply:
पेज -> पृष्ठ

Reply:
बातचीत -> interaction in hindi

Reply:
गोल यात्रा makes no sense

Reply:
revisit this once completely

Reply:
> revisit this once completely

यद्यपि आप React में सिंगल-पृष्ठ एप्लिकेशन का निर्माण कर सकते हैं,पर यह कोई आवश्यक नहीं है। React का उपयोग मौजूदा वेबसाइटस के छोटे हिस्सों में अतिरिक्त इंट्रक्टिविटी बढ़ाने के लिए भी किया जा सकता है। React में लिखा कोड PHP, या अन्य क्लाइंट-साइड लाइब्रेरीज के द्वारा सर्वर पर रेंडर किए गए मार्कअप के साथ शांतिपूर्वक सहवास कर सकता है। वास्तव में, फेसबुक में React का उपयोग ठीक इसी तरह किया जा रहा है।

Reply:
first line could be reworded as सिंगल-पृष्ठ एप्लिकेशन एक ऐसा एप्लिकेशन है जो एक HTML पृष्ठ को लोड करता है और uske saath सभी आवश्यक एसेट्स  (जैसे जावास्क्रिप्ट और css) जो एप्लिकेशन को चलाने के लिए आवश्यक है ko bhi load karta hai. 

Also, पुनः लोड -> reload in hindi

Reply:
> first line could be reworded as सिंगल-पृष्ठ एप्लिकेशन एक ऐसा एप्लिकेशन है जो एक HTML पृष्ठ को लोड करता है और uske saath सभी आवश्यक एसेट्स (जैसे जावास्क्रिप्ट और css) जो एप्लिकेशन को चलाने के लिए आवश्यक है ko bhi load karta hai.
> 
> Also, पुनः लोड -> reload in hindi

सिंगल-पृष्ठ एप्लिकेशन एक ऐसा एप्लिकेशन है जो एक HTML पृष्ठ को लोड करता है और उसके साथ सभी आवश्यक एसेट्स (जैसे जावास्क्रिप्ट और css) जो एप्लिकेशन को चलाने के लिए आवश्यक है को भी लोड करता है। पृष्ठ या बाद के पृष्ठों के साथ किसी भी इंटरेक्शन के लिए सर्वर तक राउंड ट्रिप की आवश्यकता नहीं होती है जिसका अर्थ है कि पृष्ठ रीलोड नहीं होगा।

Reply:
sounds good